src/platform/w3dengine/client/w3dbridgebuffer.cpp

```diff
@@ -0,0 +1,788 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief Bridge buffer
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dbridgebuffer.h"
+#include "assetmgr.h"
+#include "baseheightmap.h"
+#include "dx8indexbuffer.h"
+#include "dx8vertexbuffer.h"
+#include "dx8wrapper.h"
+#include "globaldata.h"
+#include "mapobject.h"
+#include "mesh.h"
+#include "meshmdl.h"
+#include "shadermanager.h"
+#include "terrainroads.h"
+#include "texture.h"
+#include "w3dshroud.h"
+#include "w3dterrainlogic.h"
+#include <cstring>
+using std::strcat;
+using std::strcpy;
+
+ShaderClass g_detailAlphaShader(361659);
+
+W3DBridge::W3DBridge() :

```

Comment:
init all other members and add a comment that its a bugfix

Reply:
fixed.